packages/docusaurus-plugin-content-docs/src/__tests__/options.test.ts

```diff
@@ -49,7 +49,9 @@ describe('normalizeDocsPluginOptions', () => {
       sidebarPath: 'my-sidebar', // Path to sidebar configuration for showing a list of markdown pages.
       sidebarItemsGenerator: DefaultSidebarItemsGenerator,
       numberPrefixParser: DefaultNumberPrefixParser,
-      docLayoutComponent: '@theme/DocPage',
+      docsRootComponent: '@theme/DocsRoot',
+      docVersionRootComponent: '@theme/DocVersionRoot',
+      docRootComponent: '@theme/DocRoot',
```

Comment:
New Root theme component we decided with @Josh-Cena .

`@theme/DocPage` becomes `@theme/DocRoot`

Hierarchy is: `@theme/DocsRoot` > `@theme/DocVersionRoot` > `@theme/DocRoot`